src/tests/ftest/launch.py

```diff
@@ -1714,6 +1714,11 @@ def resolve_debuginfo(pkg):
     return package_info
 
 
+def is_el(distro):
+    """Return True if a distro is an EL"""
+    return [d for d in ["almalinux", "rocky", "centos", "rhel"] if d in distro.name.lower()]
```

Comment:
This actually returns a `list`, not `True`. Though `bool([]) == False` and `bool([...] == True`

Reply:
`True` (see what I did there? :smile:) enough.  I could/should have included the explicit cast to be more, well, explicit.  :-)